### PR TITLE
add new methods for entities and tasks

### PR DESCRIPTION
**Pull request by pcharmoille**
**Problem**
- no method to retrieve all entities
- no method to create an entity type
- no method to create a task type
- no method to create a task status

**Solution**
add methods for all.

### CODE REVIEW

**Comment by frankrousseau** [comment]
Thank you for these additions. I just have one question about the `all_entities` function.

**Comment by pcharmoille** [comment]
> Thank you for these additions. I just have one question about the `all_entities` function.

**Comment by pcharmoille** [comment]
don't mind the close/reopen.
What do you wanted to ask about the `all_entities` function ?

**Comment by frankrousseau** [comment]
Are you sure you need that function? It will lead to huge lists.

**Comment by pcharmoille** [comment]
We use it only on our test database wich do not contains a lot of entities.
But it's up to you, I can remove it and we will call `fetch_all` directly.

File: gazu/entity.py [hunk 1]
```diff
@@ -4,6 +4,15 @@
 from .sorting import sort_by_name
 
 
+@cache
+def all_entities():
```
[the remark this answers is not in the document]
Comment (pcharmoille) [reply gazu/entity.py]: Sorry I replied outside of the thread. 

> We use it only on our test database wich do not contains a lot of entities.
But it's up to you, I can remove it and we will call fetch_all directly.

**Comment by frankrousseau** [comment]
Ok, got it. Thank you for the PR.

**Pull request merged**